pkg/plugins/plugins.go

```diff
@@ -367,7 +367,7 @@ func (s *PluginScanner) loadPlugin(pluginJSONFilePath string) error {
 	return nil
 }
 
-func (scanner *PluginScanner) IsBackendOnlyPlugin(pluginType string) bool {
+func (s *PluginScanner) IsBackendOnlyPlugin(pluginType string) bool {
```

Comment:
```suggestion
func (*PluginScanner) IsBackendOnlyPlugin(pluginType string) bool {
```

Since `s` isn't used, do we need to name the receiver?

I don't think we'll need it to be a pointer receiver either, but didn't change that since it shouldn't hurt if the PluginScanner is only ever used as a reference.

Reply:
Having a look.